runtime/CMakeLists.txt

```
@@ -120,19 +120,17 @@ install_helper(
   FILES ${CMAKE_CURRENT_SOURCE_DIR}/neovim.ico
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime)
 
-globrecurse_wrapper(RUNTIME_PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR} *.awk *.sh *.bat)
-
-foreach(PROG ${RUNTIME_PROGRAMS})
-  get_filename_component(BASEDIR ${PROG} DIRECTORY)
-  install_helper(PROGRAMS ${PROG}
-                 DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/${BASEDIR})
+glob_wrapper(RUNTIME_ROOT_FILES *.vim *.lua)
+foreach(F ${RUNTIME_ROOT_FILES})
+  install_helper(FILES ${F}
+                 DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/)
 endforeach()
```

I think we can skip the loop here

```suggestion
install_helper(FILES ${RUNTIME_ROOT_FILES}
             DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/)
```